docs/RELEASING.md

```diff
@@ -11,6 +11,12 @@
    k9mail.keyPassword=<password 'k9mail@tb-android' in 1Password>
    ```
 
+## One-time setup for F-Droid builds
+
+1. Clone the [fdroidserver](https://gitlab.com/fdroid/fdroidserver) repository and make sure the `fdroid` script is working.
```

Comment:
To ensure that `fdroid` functions correctly, the existing guidance is somewhat unclear. It would be helpful to refer to the [install instructions](https://f-droid.org/docs/Installing_the_Server_and_Repo_Tools) for more details.

On macOS you have different options on how to install `fdroidserver`. While Homebrew is an option, it may encounter issues with certain Python dependencies and the absence of `sha256sum`.

To solve the missing `sha256sum`, you can install `coreutils` by running `brew install coreutils`. However, in some setups this could lead to conflicts with pre-existing tools. In case the conflict can't be solved, the [docker image](https://gitlab.com/fdroid/docker-executable-fdroidserver) is the best alternative.

Alternatively you could clone the `fdroidserver` repository as suggested. But it requires `sha256sum` to be installed too. Also the mentioned `pyvenv` tools don't exist in current Python versions anymore, you need to use `python -m venv venv` and `source venv/bin/activate` to create a virtual environment.

Reply:
I changed the text in a32473af457697fcec35fdfbcba07af8a8107f3e